users/migrations/0003_user_role.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2 on 2021-04-13 17:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("users", "0002_auto_20210406_1328"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="user",
+            name="role",
+            field=models.CharField(
+                choices=[("user", "User"), ("tech", "Tech"), ("admin", "Admin")],
```

Comment:
Jak możesz to przypomnij co to oznacza -> w sensie po co jest z małej i wielkiej litery

Reply:
choices to zawsze lista par, w parze jest `(<wartość-wpisana-do-db>, <wartość-prezentowana-użytkownikom>)`. Czyli w tym przypadku do bazy wpisujemy string (z małej litery), ale prezentujemy (np. w django-admin) z wielkiej.
Teoretycznie moglibysmy w bazie trzymać SmallPositiveInteger i zaoszczędzić trochę miejsca, ale zależało mi na czytelności (do debugowania).